object-manipulation-1/src/to-object.js

```diff
@@ -0,0 +1,24 @@
+/* exported toObject */
+
+function toObject(keyValuePair) {
+  var newObject = {};
+  var property;
+  var value;
+  for (var i = 0; i < keyValuePair.length; i++) {
+    property = keyValuePair[0];
+    value = keyValuePair[1];
+    newObject[property] = value;
+  } return newObject;
```

Comment:
Make sure `return` statements are on their own line, they should not be on the same line as a closing brace